third_party/googletest/src/googlemock/src/gmock-cardinalities.cc

```diff
@@ -33,7 +33,9 @@
 
 #include "gmock/gmock-cardinalities.h"
 
+#if !GTEST_OS_STARBOARD
```

Comment:
Ideally we shouldn't be commenting out standard headers, but if this runs into issues with current Starboard, can leave as is.

Reply:
removed.